src/officehours_api/views.py

```diff
@@ -336,13 +335,13 @@ def get(self, request, queue_id=None, format=None):
         return response
 
     @staticmethod
-    def extract_log(queue_ids: List[int], response: HttpResponse) -> None:
+    def extract_log(queue_ids, response: HttpResponse) -> None:
```

Comment:
I may leave in a type hint of either `Any `or probably even better `Iterable[int]`. `Iterable[int]` would cover this `QuerySet` that goes through a `.values_list` call.